code/modules/projectiles/ammo_datums.dm

```diff
@@ -2711,18 +2798,66 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	name = "sniper heat bolt"
 	icon_state = "microwavelaser"
 	hud_state = "laser_heat"
-	shell_speed = 2.5
 	damage = 40
-	penetration = 0
+	penetration = 10
 	accurate_range_min = 5
 	flags_ammo_behavior = AMMO_ENERGY|AMMO_INCENDIARY|AMMO_SUNDERING|AMMO_HITSCAN|AMMO_SNIPER
 	sundering = 1
 	hitscan_effect_icon = "u_laser_beam"
 
+/datum/ammo/energy/lasgun/marine/shatter
+	name = "sniper shattering bolt"
+	icon_state = "microwavelaser"
+	hud_state = "laser_heat"
+	damage = 40
+	penetration = 30
+	accurate_range_min = 5
+	sundering = 10
+	hitscan_effect_icon = "pu_laser"
+	///number of melting stacks to apply when hitting mobvs
+	var/shatter_duration = 5 SECONDS
+
+/datum/ammo/energy/lasgun/marine/shatter/on_hit_mob(mob/M, obj/projectile/proj)
+	if(!isliving(M))
+		return
+
+	var/mob/living/living_victim = M
+	living_victim.apply_status_effect(STATUS_EFFECT_SHATTER, shatter_duration)
+
+/datum/ammo/energy/lasgun/marine/ricochet
+	name = "sniper laser bolt"
+	icon_state = "microwavelaser"
+	hud_state = "laser_heat"
+	damage = 100
+	penetration = 30
+	flags_ammo_behavior = AMMO_ENERGY|AMMO_INCENDIARY|AMMO_SUNDERING|AMMO_HITSCAN|AMMO_SNIPER
+	sundering = 1
+	hitscan_effect_icon = "u_laser_beam"
+
+/datum/ammo/energy/lasgun/marine/ricochet/one
+	damage = 80
+	bonus_projectiles_type = /datum/ammo/energy/lasgun/marine/ricochet
```

Comment:
Is this backwards? Whenever you bounce you go up a tier, as far as I can tell, but since you start out with the highest nothing ever changes.

Reply:
nah it goes down. You start with 4, its bonus projectile is 3, and so on.

Reply:
Oh you do start with four. Weird but alright.

Reply:
its because the bonus proj would be inherited if you start with 0, so you'd have to override it on 4. save muh 1 line.